crates/storage_impl/src/payments/payment_intent.rs

```diff
@@ -744,39 +744,40 @@ impl<T: DatabaseStore> PaymentIntentInterface for crate::RouterStore<T> {
     ) -> error_stack::Result<Vec<(common_enums::IntentStatus, i64)>, StorageError> {
         let conn = connection::pg_connection_read(self).await.switch()?;
         let conn = async_bb8_diesel::Connection::as_async_conn(&conn);
-
+    
```

Comment:
Hey @ayan-joshi why these lines are changes and there's extra space at start?

Reply:
> Hey @ayan-joshi why these lines are changes and there's extra space at start?

@apoorvdixit88  I guess one line changed from above the fucnction so it is there for every line , does it affect the code ?

Reply:
@ayan-joshi Please address the failing checks.